packages/entities/entities-shared/src/composables/useFetcher.ts

```diff
@@ -121,5 +124,21 @@ export default function useFetcher(
     }
   }
 
-  return { fetcher, fetcherState: state }
+  const cacheId = config.cacheIdentifier
+  const fetcherCacheKey = ref<number>(1)
+  if (cacheId) {
+    if (cacheKeys.has(cacheId)) {
+      fetcherCacheKey.value = cacheKeys.get(cacheId)!
+    } else {
+      cacheKeys.set(cacheId, fetcherCacheKey.value)
+    }
+  }
+
+  watch(fetcherCacheKey, () => {
+    if (cacheId) {
+      cacheKeys.set(cacheId, fetcherCacheKey.value)
```

Comment:
nit: you should access via the param rather than the ref directly
```suggestion
  watch(fetcherCacheKey, (k) => {
    if (cacheId) {
      cacheKeys.set(cacheId, k)
```

Reply:
Nice catch! Updated.